lib/msal-common/src/request/BaseAuthRequest.ts

```diff
@@ -43,4 +44,5 @@ export type BaseAuthRequest = {
     maxAge?: number;
     tokenQueryParameters?: StringDict;
     storeInCache?: StoreInCache;
+    reqCnf?: ReqCnfData;
```

Comment:
Looks like `kid` is needed for the native broker case, else we could just make this a `string`.

Reply:
Not sure why it would be needed in addition to reqCnf - let's confirm that

Reply:
I will follow up on this in a separate PR.